pkgs/tools/games/minecraft/packwiz/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGo118Module
+, fetchFromGitHub
+}:
+
+buildGo118Module rec {
+  pname = "packwiz";
+  version = "unstable-2022-5-25";
+
+  src = fetchFromGitHub {
+    owner = "packwiz";
+    repo = "packwiz";
+    rev = "e71b63ea98283c8c1f0e03ee51ae40f452f22a61";
+    sha256 = "sha256-XwGacEVfQAduDCSMQFRw7Xnx4bND2zaV7l27B+2u5xg=";
+  };
+
+  vendorSha256 = "sha256-M9u7N4IrL0B4pPRQwQG5TlMaGT++w3ZKHZ0RdxEHPKk=";
+
+  meta = with lib; {
+    description = "A command line tool for editing and distributing Minecraft modpacks, using a git-friendly TOML format.";
```

```suggestion
    description = "A command line tool for editing and distributing Minecraft modpacks, using a git-friendly TOML format";
```
[Small nitpick](https://github.com/NixOS/nixpkgs/blob/c46c3ccefa2013392ee41fd846ff7e03658bbc2b/CONTRIBUTING.md?plain=1#L43)